src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagReporter.java

```diff
@@ -24,11 +24,7 @@
      */
     public static void initializeBugsnagErrorReporting() {
         String apiKey = getConfigPropertyAsText("BUGSNAG_PROJECT_NOTIFIER_API_KEY");
-        if (apiKey != null) {
-            bugsnag = new Bugsnag(apiKey);
-        } else {
-            LOG.warn("Bugsnag project notifier API key not available. Bugsnag error reporting disabled.");
-        }
+        bugsnag = new Bugsnag(apiKey);
```

Comment:
Yea, so for this one, I think we just make the API key optional. Apologies for making the original rec to to make it required.

Reply:
Updated.